### PR TITLE
[REVIEW] Cybert Streamz Workflow

### PR DESCRIPTION
**Pull request by brhodes10**
This PR updates the cyBERT streamz workflow. A docker file and script is provided which can automatically run a demonstration of cyBERT with streamz and Kafka, given a certain set of parameters like model, labels, etc.

*This PR is in coordination with the latest cybert class PR #189

### CODE REVIEW

**Comment by brhodes10** [comment]
rerun tests

**Comment by brhodes10** [comment]
Updating docker file to align with new clx docker updates

**Comment by brhodes10** [comment]
rerun tests

**Comment by brhodes10** [comment]
rerun tests

**Comment by brhodes10** [comment]
rerun tests

File: examples/streamz/Dockerfile [hunk 1]
```diff
@@ -1,54 +1,57 @@
-ARG RAPIDS_VERSION=0.13
-ARG CUDA_VERSION=10.1
+# An integratio:n test & dev container which builds and installs CLX from default branch
+ARG CUDA_VERSION=10.2
+ARG CUDA_SHORT_VERSION=${CUDA_VERSION}
 ARG LINUX_VERSION=ubuntu18.04
 ARG PYTHON_VERSION=3.7
+FROM gpuci/miniconda-cuda:${CUDA_VERSION}-devel-${LINUX_VERSION}
```
Comment (efajardo-nv) [inline examples/streamz/Dockerfile:6]: just fyi we can go back to rapidsai-dev-nightly image since blas requirements have been relaxed.

**Pull request merged**